libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ModelOverviewChartPivot.tsx

```diff
@@ -192,6 +194,25 @@ export class ModelOverviewChartPivot extends React.Component<
               <ConfusionMatrixHeatmap />
             </PivotItem>
           )}
+        {!ifEnableLargeData(this.context.dataset) &&
+          (IsBinary(this.context.modelMetadata.modelType) ||
+            IsMulticlass(this.context.modelMetadata.modelType)) && (
+            // isFlightActive(
+            //   aucChartExperienceFlight,
+            //   this.context.featureFlights
+            // ) && (
+            // TODO: support multiclass: IsMulticlass(this.context.modelMetadata.modelType)) && (
```

Comment:
I guess you would need a drop-down to select the one vs all class and then generate the 0s and 1s based on that.

Reply:
i think the idea would be to show a line for each class; does that make sense ? the user can optionally disable the line's visibility (see the roc chart here [https://ml.azure.com/experiments/id/e01fdc58-9702-4fb8-add9-1a5d09b21058/runs/iris-automl-1_24?wsid=/subscriptions/e0fd569c-e34a-4249-8c24-e8d723c7f054/resourceGroups/hawestra-rg/providers/Microsoft.MachineLearningServices/workspaces/hawestra-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#metrics](url)

Reply:
That will be tricky if you have one line up per cohort as well. Or are we showing just for current cohort?